[2024-01-31T11:57:22.032+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: stock_analysis_dag.perform_descriptive_analysis manual__2024-01-31T11:57:08.360636+00:00 [queued]>
[2024-01-31T11:57:22.036+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: stock_analysis_dag.perform_descriptive_analysis manual__2024-01-31T11:57:08.360636+00:00 [queued]>
[2024-01-31T11:57:22.036+0000] {taskinstance.py:2171} INFO - Starting attempt 1 of 3
[2024-01-31T11:57:22.046+0000] {taskinstance.py:2192} INFO - Executing <Task(PythonOperator): perform_descriptive_analysis> on 2024-01-31 11:57:08.360636+00:00
[2024-01-31T11:57:22.049+0000] {standard_task_runner.py:60} INFO - Started process 2918 to run task
[2024-01-31T11:57:22.051+0000] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'stock_analysis_dag', 'perform_descriptive_analysis', 'manual__2024-01-31T11:57:08.360636+00:00', '--job-id', '35', '--raw', '--subdir', 'DAGS_FOLDER/stock_analysis_dag.py', '--cfg-path', '/tmp/tmp242ey_hx']
[2024-01-31T11:57:22.052+0000] {standard_task_runner.py:88} INFO - Job 35: Subtask perform_descriptive_analysis
[2024-01-31T11:57:22.072+0000] {task_command.py:423} INFO - Running <TaskInstance: stock_analysis_dag.perform_descriptive_analysis manual__2024-01-31T11:57:08.360636+00:00 [running]> on host de-airflow-vm-instance-1.europe-west10-a.c.dataengineeringfinal-411911.internal
[2024-01-31T11:57:22.111+0000] {taskinstance.py:2481} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='onatpostac@gmail.com' AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='stock_analysis_dag' AIRFLOW_CTX_TASK_ID='perform_descriptive_analysis' AIRFLOW_CTX_EXECUTION_DATE='2024-01-31T11:57:08.360636+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-01-31T11:57:08.360636+00:00'
[2024-01-31T11:57:22.125+0000] {connection.py:234} WARNING - Connection schemes (type: google_cloud_platform) shall not contain '_' according to RFC3986.
[2024-01-31T11:57:22.126+0000] {base.py:83} INFO - Using connection ID 'google_cloud_default' for task execution.
[2024-01-31T11:57:22.519+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-01-31T11:57:22.522+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=stock_analysis_dag, task_id=perform_descriptive_analysis, execution_date=20240131T115708, start_date=20240131T115722, end_date=20240131T115722
[2024-01-31T11:57:22.543+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-01-31T11:57:22.551+0000] {taskinstance.py:3281} INFO - 0 downstream tasks scheduled from follow-on schedule check
